### ISSUE TITLE
infracost --terraform-plan-flags does not work with multiple -var-file

### ISSUE THREAD

**Issue by anhdle14** [antonbabenko/pre-commit-terraform#279]
### Describe the bug

The --terraform-plan-flags inside pre-commit does not work with multiple -var-file arguments. Here is the gist of the `.pre-commit-config.yaml`

```yaml
repos:
  - repo: git://github.com/antonbabenko/pre-commit-terraform
    rev: v1.56.0
    hooks:
      - id: infracost_breakdown
        args:
          - --args=--path=. --terraform-plan-flags="-var-file=_default/terraform.tfvars -var-file=$ENVIRONMENT/terraform.tfvars" --show-skipped
```

Here is the command that works if run normally:

```bash
$ infracost breakdown --terraform-plan-flags "-var-file=$ENVIRONMENT/terraform.tfvars -var-file=_default/terraform.tfvars" --path live --show-skipped   
```

Here is the output from running via pre-commit:
```bash
~/Developer/github.com/jeraoss/terraform-iaas-template main* ❯ pre-commit run -a -c .pre-commit-ci.yaml                                                                                                                            19:28:46
Infracost breakdown......................................................Failed
- hook id: infracost_breakdown
- duration: 1.01s
- exit code: 1

Error: unknown shorthand flag: 'v' in -var-file=$ENVIRONMENT/terraform.tfvars"
```

### How can we reproduce it?

1. Passing multiple -var-file inside --terraform-plan-flags.

### Environment information

* OS:  `Darwin MacBook.local 21.2.0 Darwin Kernel Version 21.2.0: Wed Oct 20 00:23:27 PDT 2021; root:xnu-8019.60.40.0.1~26/RELEASE_X86_64 x86_64`

* Tools availability and versions:

pre-commit: `2.15.0`
python: `2.7.18`
python3: `3.9.7`
infracost: `0.9.12`

<details><summary>.pre-commit-config.yaml</summary>

```yaml
repos:
  - repo: git://github.com/antonbabenko/pre-commit-terraform
    rev: v1.56.0
    hooks:
      - id: infracost_breakdown
        args:
          - --args=--path=. --terraform-plan-flags="-var-file=_default/terraform.tfvars -var-file=$ENVIRONMENT/terraform.tfvars" --show-skipped
```

</details>

**Comment by anhdle14** [comment]
@MaxymVlasov both of the codes I have tried, and return the same error: 

```bash
Error: unknown shorthand flag: 'v' in -var-file=$ENVIRONMENT/terraform.tfvars"
```

> Also, `$ENVIRONMENT` from `-var-file=$ENVIRONMENT/terraform.tfvars` will not work (see YAML spec for details), you should specify the path.

Thank you, I will remove it from the file.

**Comment by MaxymVlasov** [comment]
```yaml
repos:
- repo: https://github.com/antonbabenko/pre-commit-terraform
  rev: v1.67.0
  hooks:
    - id: infracost_breakdown
      args:
        - --args=--path=./environment/qa
        - --args=--terraform-plan-flags="-var-file=../../test.tfvars"
        - --args=--terraform-plan-flags="-var-file=../../test2.tfvars"
```

That works as expected. I do nothing in codebase, so in v1.56.0 it should works too

**Comment by MaxymVlasov** [comment]
Can't find the root cause of what goes wrong in this command:

https://github.com/antonbabenko/pre-commit-terraform/blob/c5462f6bf9a4cac24b52fde3ec567424246cbb71/hooks/infracost_breakdown.sh#L45

Maybe, some weird interference with infracost program logic, etc.
I'll document that situation.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 1.68.0 :tada:
